ash-window/Cargo.toml

```diff
@@ -18,13 +18,26 @@ rust-version = "1.64.0"
 ash = { path = "../ash", version = "0.37", default-features = false }
 raw-window-handle = "0.5"
 
+[target.'cfg(unix)'.dependencies]
+x11-dl = { version = "2.21.0", optional = true }
+x11 = { version = "2.21.0", features = ["xlib"], optional = true }
+xcb = { version = "1.2.2", optional = true }
+
 [target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies]
 raw-window-metal = "0.3"
 
 [dev-dependencies]
 winit = "0.28.0"
 ash = { path = "../ash", version = "0.37", default-features = false, features = ["linked"] }
+xcb = { version = "1.2.2" }
+
+[features]
+default = ["x11-dl", "xcb"]
```

Comment:
Requiring `xcb` by default means we need libxcb present at buildtime, right? While probably most environments will have this, disabling it will make it easier for downstream users to get CI working in minimal VMs and such.

Reply:
The reason for defaulting this is to have the least surprising behavior by default. I dont think the default should be to not support a very broadly used platform.

Reply:
We know winit doesn't use xcb. Where in the raw-window-handle ecosystem is xcb broadly used?

Reply:
Good point, however if winit at some point decides to switch to xcb, it should be defaulted again...

Reply:
I think winit will want to avoid adding foreign build-time dependencies for the same reason.